src/components/conjunction/tables/ConjunctionEventSummaryTable.tsx

```diff
@@ -40,16 +40,19 @@ const ConjunctionEventSummaryTableInformationsTable = ({ object }: ConjunctionEv
     accessorKey: 'missDistance',
   }, {
     header: t('summary_list.radial_miss_distance'),
+    renderCell: row => row.radialMissDistance && Math.abs(row.radialMissDistance),
```

Comment:
create util for this probabiltywe will use it in the future

Reply:
fixed